shortfin/python/shortfin_apps/llm/components/invocation.py

```diff
@@ -443,23 +440,23 @@ def __init__(
         self,
         name: str,
         fiber: sf.Fiber,
-        array_cache: DeviceArrayCache,
         llm_task: LlmTask,
         functions: dict[int, sf.ProgramFunction],
-        seq_stride: int,
         page_tables,
         program_isolation: sf.ProgramIsolation,
+        completion_callback: Callable,
```

Comment:
See this just defines a callable function with no for what the callback should look like. You need to define something at the LlmInvoker level that is overridden or there is no way for a user to implement.